src/BenchmarkDotNet.Core/Columns/IColumn.cs

```diff
@@ -16,8 +16,30 @@ public interface IColumn
         /// </summary>
         string ColumnName { get; }
 
+        /// <summary>
+        /// Column title formatted using the specified style.
+        /// </summary>
+        /// <param name="style"></param>
+        /// <returns></returns>
+        string GetName(ISummaryStyle style);
+
+        /// <summary>
+        /// Value in this column formatted using the default style.
+        /// </summary>
+        /// <param name="summary"></param>
```

Comment:
Can I remove `<param>` and `<returns>` leaving just the `<summary>`? There was no documentation for this method before, and I'm afraid the comments will be similar to [this](https://github.com/dotnet/BenchmarkDotNet/pull/379#discussion_r104139993)